src/main/java/seedu/address/logic/commands/SelectCommand.java

```diff
@@ -47,16 +51,16 @@ public SelectCommand(Index targetIndex, int selectType) {
     public CommandResult execute(Model model, CommandHistory history) throws CommandException {
```

Comment:
This methods seems to be too long. Consider extracting the processor for each types of command outside